cmd/swarm/config_test.go

```diff
@@ -302,6 +302,112 @@ func TestConfigCmdLineOverrides(t *testing.T) {
 	node.Shutdown()
 }
 
+func TestSwapConfigCmdLineOverrides(t *testing.T) {
```

Comment:
This aims to test the flag functionality of swap logs.
The problem when this executed is the backend the ipc is not found
`config_test.go:359: dial unix /tmp/bzztest125129753/bzzd.ipc: connect: no such file or directory`

There is other test in swap_test.go but they do not invoque the flags, but the ipc works.
Maybe there is a reason the flags were not tested.

Any feedback would be great.
@Eknir @mortelli @ralph-pichler

Reply:
the directory seems to be correctly created when executing this test, but the .ipc file does not exist.

take a look at other tests which call `rpc.Dial` based on the given IPC endpoint parameter. 

see if you can find the code before this call which creates the (in this case missing) .ipc file.

Reply:
The test is not working correctly with swap left an issue for further discussion #2127
The flags are tested manually and are working accordingly.
 
I personally think we can merge this PR.